docs/_i18n/en.yml

```diff
@@ -0,0 +1,27 @@
+langs:
```

Comment:
Is this a generated directory? Or do we have to keep this updated with translations?

Reply:
This is not autogenerated. If we add a new page **and** we want it to be "translatable", the page should use the `i18n_page` layout and we need to add the appropriate title and / or content to this directory. If we make it an `i18n_page` and forget to add stuff to the `en.yml` or `en` directory the build will fail, so that part is enforced.